# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/kde/kdeadmin/libtool-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


A tiny "hack" for our /lib vs. /usr/lib situation. Too damn kde* uses an
own copy of libtool - we need to verify how the official GNU beast behaves
and maybe really fix that one.

  - Rene Rebe <rene@rocklinux.org>

--- kdeadmin-3.1.4/admin/ltmain.sh.orig	2003-12-03 16:30:02.000000000 +0100
+++ kdeadmin-3.1.4/admin/ltmain.sh	2003-12-03 16:30:56.000000000 +0100
@@ -1963,9 +1963,12 @@
 	    dir="$ladir"
 	    absdir="$abs_ladir"
 	    libdir="$abs_ladir"
-	  else
+	  elif test -f "$libdir/$linklib" ; then
 	    dir="$libdir"
 	    absdir="$libdir"
+	  else
+	    dir="/lib"
+	    absdir="/lib"
 	  fi
 	else
 	  dir="$ladir/$objdir"
